phoenix-core/src/main/java/org/apache/phoenix/query/QueryServicesOptions.java

```diff
@@ -223,11 +223,13 @@ public class QueryServicesOptions {
      * HConstants#HIGH_QOS is the max we will see to a standard table. We go higher to differentiate
      * and give some room for things in the middle
      */
+    public static final int DEFAULT_SERVER_SIDE_PRIORITY = 500;
```

Comment:
@jpisaac - I'm curious how you determined this priority (much less than metadata, which is very similar, and less than indexes)

Reply:
The priority is only used to dispatch to the right RPCExecutor. In this case, since the index, metadata and server-server use separate Executors not sure whether the actual value matters?